code/modules/surgery/tools.dm

```diff
@@ -111,18 +111,8 @@
 	if(contents.len)
 		to_chat(user, "<span class='notice'>[src] already has something inside it.</span>")
 		return
-	if(isorgan(I))
-		var/obj/item/organ/O = I
-		if(O.status != ORGAN_ORGANIC)
-			to_chat(user, "<span class='notice'>[src] can only hold organic body parts!</span>")
-			return
-	else if(isbodypart(I))
-		var/obj/item/bodypart/BP = I
-		if(BP.status != BODYPART_ORGANIC)
-			to_chat(user, "<span class='notice'>[src] can only hold organic body parts!</span>")
-			return
-	else
-		return
+	if(!isorgan(I) || !isbodypart(I))

```

Comment:
`if(!isorgan(I) && !isbodypart(I))`
 **(tab here)**`return`

Logic:
If the object is not an organ, check to see if it is a body part. If it is NOT an organ and it is NOT a bodypart, then return.

Additional recommendation: Add `to_chat(user, "<span class='notice'>[src] can only hold organic body parts!</span>")` on line 115 (under the if check), make sure it has two tabs (so that it is controlled by the if statement). Put the return right under it (also with two tabs)

It will look a bit like this if you are using DM, highlighted for your convenience. The `>>` you see are tabs.
![image](https://cloud.githubusercontent.com/assets/4955510/25776062/9b6f0538-3279-11e7-9d2b-e7e38ab24bbb.png)


Sorry about the bad code earlier - I did not give it more than a few seconds of thought.